src/screens/ProfileScreen/LanguageScreen.js

```diff
@@ -82,14 +82,53 @@ const styles = StyleSheet.create({
     color: '#F55963',
   },
   wheelPicker: {
+    // backgroundColor: '#ccc',
+    height: 270,
     width: '100%',
-    flex: 1,
+    // flex: 1,
     alignItems: 'center',
   },
+  itemStyleVertical: {
+    width: 150,
+    height: 50,
+    borderRadius: 10,
+    justifyContent: 'center',
+  },
+
+  itemSelectedStyleVertical: {
+    borderColor: '#fff',
+  },
+
+  itemSelectedStyleVertical_: {
+    backgroundColor: '#F55963',
+  },
 });
 
+const Bubble = props => {
+  const { children, selected } = props;
+  return (
+    <View
+      style={[
+        styles.itemStyleVertical,
+        selected ? styles.itemSelectedStyleVertical_ : styles.itemSelectedStyleVertical,
+      ]}
+    >
+      <Text
+        style={{
+          textAlign: 'center',
+          fontSize: selected ? 20 : 17,
+          color: selected ? 'white' : 'gray',
+          fontWeight: selected ? 'bold' : 'normal',
+        }}
+      >
+        {children}
+      </Text>
+    </View>
+  );
+};
+
 const LanguageScreen = () => {
-  const [selectedLanguageIndex, setSelectedLanguageIndex] = React.useState(0);
+  const [selectedLanguageIndex, setSelectedLanguageIndex] = React.useState(2);
```

Comment:
Please set this to `0`

Reply:
Was Setting it to 2 so that the English would be default language